gateway-js/src/__tests__/loadServicesFromRemoteEndpoint.test.ts

```diff
@@ -10,6 +10,7 @@ describe('getServiceDefinitionsFromRemoteEndpoint', () => {
       getServiceDefinitionsFromRemoteEndpoint({
         serviceList,
         serviceSdlCache,
+        getServiceIntrospectionHeaders: () => ({})
```

Comment:
In current form, these should be `async` but I'm suggesting a change below that allows them to return a `Promise` or value so shouldn't be a change needed here.